go-controller/pkg/node/routemanager/route_manager.go

```diff
@@ -259,7 +259,7 @@ func (c *Controller) sync() {
 			filterRoute, filterMask := filterRouteByDstAndTable(linkIndex, managedRoute.Dst, managedRoute.Table)
 			existingRoutes, err := util.GetNetLinkOps().RouteListFiltered(netlink.FAMILY_ALL, filterRoute, filterMask)
 			if err != nil {
-				klog.Errorf("Route Manager: failed to list routes for link  %w", filterRoute.String(), filterMask, linkIndex, err)
+				klog.Errorf("Route Manager: failed to list routes for link  %s/%v/%v: %v", filterRoute.String(), filterMask, linkIndex, err)
```

Comment:
a bit cryptic and fix spacing chance

`Route Manager: failed to list routes for link %s with route filter %s and mask filter %v: %v`

Reply:
gracias! copied this :D saves me the trouble of computing all the fillers :D